onos-ztp/values.yaml

```diff
@@ -0,0 +1,39 @@
+# Default values for onos-ztp.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: onosproject/onos-ztp
+  tag: latest
+  pullPolicy: IfNotPresent
+  pullSecrets: []
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: "onos-ztp"
+
+debug: false
+
+store:
+  enabled: true
+  raftgroup: onos-ztp-raft
```

Comment:
does ZTP depend on Atomix? What does this do?

Reply:
No it does not I was thinking we can keep it in case we want to change the store to atomix but I will remove it for now.